debian/schema-registry/include/etc/confluent/docker/configure

```diff
@@ -19,7 +19,11 @@ set -o nounset \
     -o verbose \
     -o xtrace
 
-dub ensure SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL
+if [[ -z "${SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL-}" && -z "${SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS-}" ]]
+then
+    dub ensure SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL
+fi
+
```

Comment:
Could you please add a line of documentation to describe what this block is trying to achieve?

Reply:
added